### ISSUE TITLE
[BUG]CRC start takes hours to start. This ran successfully before i removed everything and tried to setup again.

### ISSUE THREAD

**Issue by gganeshan2014** [crc-org/crc#3611]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop 

## CRC version
```bash
# Put `crc version` output here
```
 C:\Users\ganes>crc version
WARN A new version (2.17.0) has been published on https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/2.17.0/crc-windows-installer.zip
CRC version: 2.15.0+72256c3c
OpenShift version: 4.12.5
Podman version: 4.3.1

## CRC status
```bash
# Put `crc status --log-level debug` output here
```
C:\Users\ganes>crc status
Machine does not exist. Use 'crc start' to create it

## CRC config
```bash
# Put `crc config view` output here
```
C:\Users\ganes>crc config view
- consent-telemetry                     : no

## Host Operating System 
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```
C:\Users\ganes>systeminfo

Host Name:                 LAPTOP-7F6TN17S
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22621 N/A Build 22621
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          ganeshan.g2014@yahoo.com
Registered Organization:   N/A
Product ID:                00330-52963-32470-AAOEM
Original Install Date:     10/8/2022, 2:52:42 AM
System Boot Time:          4/24/2023, 12:48:06 PM
System Manufacturer:       LENOVO
System Model:              20R1S0M000
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 142 Stepping 12 GenuineIntel ~1609 Mhz
BIOS Version:              LENOVO N2QET46W(1.40 ), 3/22/2022
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+05:30) Chennai, Kolkata, Mumbai, New Delhi
Total Physical Memory:     16,139 MB
Available Physical Memory: 9,240 MB
Virtual Memory: Max Size:  28,427 MB
Virtual Memory: Available: 20,709 MB
Virtual Memory: In Use:    7,718 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\LAPTOP-7F6TN17S
Hotfix(s):                 4 Hotfix(s) Installed.
                           [01]: KB5022497
                           [02]: KB5012170
                           [03]: KB5025239
                           [04]: KB5025749
Network Card(s):           7 NIC(s) Installed.
                           [01]: TAP-Windows Adapter V9
                                 Connection Name: Local Area Connection
                                 Status:          Hardware not present
                           [02]: Intel(R) Wireless-AC 9560 160MHz
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.15
                                 [02]: fe80::6173:aaba:59b:a163
                                 [03]: 2401:4900:1c97:46b1:24e7:1224:68da:b340
                                 [04]: 2401:4900:1c97:46b1:e534:8c72:e0b6:c073
                           [03]: Intel(R) Ethernet Connection (10) I219-V
                                 Connection Name: Ethernet
                                 Status:          Media disconnected
                           [04]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection 2
                                 Status:          Media disconnected
                           [05]: VMware Virtual Ethernet Adapter for VMnet1
                                 Connection Name: VMware Network Adapter VMnet1
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.64.254
                                 IP address(es)
                                 [01]: 192.168.64.1
                                 [02]: fe80::cf09:255e:5a65:858d
                           [06]: VMware Virtual Ethernet Adapter for VMnet8
                                 Connection Name: VMware Network Adapter VMnet8
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.180.254
                                 IP address(es)
                                 [01]: 192.168.180.1
                                 [02]: fe80::c0d4:3786:99f8:427d
                           [07]: VirtualBox Host-Only Ethernet Adapter
                                 Connection Name: Ethernet 2
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.56.1
                                 [02]: fe80::83ca:ffbf:97df:e664
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
### Steps to reproduce

CRC start hanging for hours

########################################################################################
C:\Users\ganes>crc start
WARN A new version (2.17.0) has been published on https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/2.17.0/crc-windows-installer.zip
INFO Checking if running in a shell with administrator rights
INFO Checking Windows release
INFO Checking Windows edition
INFO Checking if Hyper-V is installed and operational
INFO Checking if Hyper-V service is enabled
INFO Checking if crc-users group exists
INFO Checking if current user is in crc-users and Hyper-V admins group
INFO Checking if vsock is correctly configured
INFO Checking if the daemon task is installed
INFO Checking if the daemon task is running
INFO Checking admin helper service is running
INFO Loading bundle: crc_hyperv_4.12.5_amd64...
CRC requires a pull secret to download content from Red Hat.
You can copy it from the Pull Secret section of https://console.redhat.com/openshift/create/local.
? Please enter the pull secret *****************************************************************************************INFO Creating CRC VM for openshift 4.12.5...
Error creating machine: Error in driver during machine creation: exit status 1

C:\Users\ganes>crc start --log-level debug
DEBU CRC version: 2.15.0+72256c3c
DEBU OpenShift version: 4.12.5
DEBU Podman version: 4.3.1
DEBU Running 'crc start'
DEBU Total memory of system is 16922796032 bytes
WARN A new version (2.17.0) has been published on https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/2.17.0/crc-windows-installer.zip
DEBU Checking file: C:\Users\ganes\.crc\machines\crc\.crc-exist
INFO Checking if running in a shell with administrator rights
DEBU Running '$currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent());$currentPrincipal.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)'
INFO Checking Windows release
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
INFO Checking Windows edition
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
DEBU Running on Windows Professional edition
INFO Checking if Hyper-V is installed and operational
DEBU Running '@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent'
DEBU Running '@(Get-Service vmms).Status'
INFO Checking if Hyper-V service is enabled
DEBU Running '@(Get-Service vmms).Status'
INFO Checking if crc-users group exists
DEBU Running 'Get-LocalGroup -Name crc-users'
INFO Checking if current user is in crc-users and Hyper-V admins group
DEBU Running 'Get-LocalGroupMember -Group 'crc-users' -Member 'ganes''
DEBU Running 'Get-LocalGroupMember -SID 'S-1-5-32-578' -Member 'ganes''
INFO Checking if vsock is correctly configured
DEBU Running 'Get-Item -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Virtualization\GuestCommunicationServices\00000400-FACB-11E6-BD58-64006A7986D3"'
INFO Checking if the daemon task is installed
DEBU Running 'Get-ScheduledTask -TaskName crcDaemon'
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").Version'
INFO Checking if the daemon task is running
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").State'
INFO Checking admin helper service is running
DEBU Running '(Get-Service crcAdminHelper).Status'
DEBU Checking file: C:\Users\ganes\.crc\machines\crc\.crc-exist
DEBU Copying 'C:\Users\ganes\.crc\cache\crc_hyperv_4.12.5_amd64\oc.exe' to 'C:\Users\ganes\.crc\bin\oc\oc.exe'
DEBU Copying 'C:\Users\ganes\.crc\cache\crc_hyperv_4.12.5_amd64\podman.exe' to 'C:\Users\ganes\.crc\bin\oc\podman.exe'
INFO Loading bundle: crc_hyperv_4.12.5_amd64...
DEBU Cannot load secret from configuration: empty path
DEBU Using secret from keyring
INFO Creating CRC VM for openshift 4.12.5...
DEBU Running pre-create checks...
DEBU Running '@(Get-Module -ListAvailable hyper-v).Name | Get-Unique'
DEBU Running '@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(([System.Security.Principal.SecurityIdentifier]::new("S-1-5-32-578")))'
DEBU Creating machine...
DEBU Creating VM...
DEBU Running 'Hyper-V\New-VM crc -Path 'C:\Users\ganes\.crc\machines\crc' -MemoryStartupBytes 9216MB'

########################################################################################

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
@gganeshan2014 As per following logs looks like crc is not even able to start. Do you have some debug logs which you can share?

```
? Please enter the pull secret *****************************************************************************************INFO Creating CRC VM for openshift 4.12.5...
Error creating machine: Error in driver during machine creation: exit status 1
```

**Comment by KulaGGin** [comment]
I'm researching a different "long start" problem for a different project. It's an overlay program for a game. It takes about 175-180 seconds to execute a single function. It's the [D3D11CreateDeviceAndSwapChain](https://learn.microsoft.com/en-us/windows/win32/api/d3d11/nf-d3d11-d3d11createdeviceandswapchain) function.

Why am I telling you this? Because the user that has this problem has these 2 updates installed: [03]: KB5025239 [04]: KB5025749. They removed the KB5025239 update and the problem still persists. They didn't have this problem a month ago, and the KB5025749 update was installed on the 13th of April(less than a month ago). And also the code that is ran during initialization and causes the problem didn't change between then and now. In fact, this code stayed the same since 2013 and is a default D3D11 initialization routing: create window class, create window, create device and swap chain, create device context. So it's most probably Windows update. They can't remove the update with `wusa /uninstall /kb:KB5025749` in the command prompt  to test, they get a `Servicing Stack 10.0.22621.1550 is required by your computer and cannot be uninstalled` error:
![image](https://i.imgur.com/I6H95D2.png)

Nice. Real nice, Microsoft.

Is this the same problem that you're having here? I don't know, my guess is that it's "probably": new issue, long start, in both cases the KB5025239 is installed.

**Comment by praveenkumar** [comment]
@gganeshan2014 Can you try to use latest version of OpenShift local and create a separate issue if still exist?
